### PR TITLE
Generic styles

### PR DESCRIPTION
**Pull request by sahil-at-home1**
Updated the `styles.css` file to have generic colors (stored in css variables) and fonts to use throughout the website. It also includes css specific to the navbar, which should be the same for each page of the website.

The index and people pages were updated to use this styles sheet first, then their unique style sheets after. This is so the unique one can override any of the defaults.

Screenshots are included below.
image
image

### CODE REVIEW

File: people.html [hunk 1]
```diff
@@ -8,24 +8,32 @@
...
+				<li class="nav-item">
+					<i class="fa fa-search search-icon"></i>
+				</li>
+			</ul>
+		</div>
+	</nav>
+
     <main class="container style-fix">
         <h1>Software</h1>
         <div class="container">
```
Comment (ilopez5) [inline people.html]: I mentioned this in PR #13, but could you change some div tags to section tags wherever it makes sense? Also, is there supposed to be two container classes used here?

**Review by ilopez5 — CHANGES_REQUESTED** [review]
Looks good, thanks for making the color variables!! See my comments

File: people.html (hunk 1 above)
Comment (sahil-at-home1) [reply people.html]: When should I use section tags instead of div tags? Also, I'm not sure about the container classes.

**Comment by saugusth** [comment]
@sahilatIIT you did an amazing job

**Comment by lehongson1912** [comment]
Looking good Sahil

**Pull request merged**